### PR TITLE
Fix: Add bail option to build:prod so that CI builds fail when they should

### PR DESCRIPTION
**Pull request by jhuntoo**
My CI build just deployed a mangled version of angular2-webpack-starter because `npm run build:prod` exits with 0 when a loader is missing in node_modules.

This PR adds the --bail option which will cause the webpack build process to exit early, and importantly exit with a non-zero status.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "watch:prod": "webpack --watch --config webpack.prod.config.js --progress --profile --colors --display-error-details --display-cached",
     "build": "npm run build:dev",
     "prebuild:dev": "npm run clean:dist",
-    "build:dev": "webpack --progress --profile --colors --display-error-details --display-cached",
+    "build:dev": "webpack --progress --profile --colors --display-error-details --display-cached --bail",
```
Comment (samvloeberghs) [inline package.json]: better to put it on the build:prod task than :p

**Comment by jhuntoo** [comment]
@samvloeberghs Derp...  thanks, made the fix in different branch!

**Comment by samvloeberghs** [comment]
and now squash :p

**Comment by gdi2290** [comment]
you should squash it but I'll just merge it. thanks!

**Pull request merged**